test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/RazorWebSite.UrlResolution.Index.html

```diff
@@ -9,12 +9,12 @@
     <link href="/Person/John Doe/CSS" rel="stylesheet">
     <video poster="/SomeUrl" src="/SomeUrl/video" />
     <audio src="/SomeUrl">
-        <source src="/Person">
+        <source src="/Person" srcset="/Person">
         <track src="/email~/SomeUrl">
     </audio>
     <embed src="/email@dyanmicUrl">
     <iframe src="/SomeUrl" />
-    <img src="/~/SomeUrl">
+    <img src="/~/SomeUrl" srcset="/~/SomeUrl">
```

Comment:
Suggest two variables e.g. `tildePath` and `relativePath` to avoid this invalidity. Would still be testing the different code paths but without veering into something no user would do.